Tools/scripts/install-prereqs-ubuntu.sh

```diff
@@ -14,6 +12,15 @@ BEBOP_PKGS="g++-arm-linux-gnueabihf"
 UBUNTU64_PKGS="libc6:i386 libgcc1:i386 gcc-4.9-base:i386 libstdc++5:i386 libstdc++6:i386"
 ASSUME_YES=false
 
+if grep -q "Ubuntu 16" /etc/os-release
+then
+	# Packages for Ubuntu 16.x
+	SITL_PKGS="g++ python-pip python-matplotlib python-serial python-wxgtk3.0 python-scipy python-opencv python-numpy python-pyparsing ccache realpath"
```

Comment:
I would prefer that we left the equal packages general and then just add the different packages with `+=`